src/shared/components/InfoButton/InfoButton.tsx

```diff
@@ -7,7 +7,7 @@ import './InfoButton.scss';
 
 interface InfoButtonProps {
   caption?: string;
-  path: RoutesPath;
+  path: RoutesPath | string;
```

Comment:
Yet I do not find a solution how to type dynamically changed root for now because it will be always changing because of id